.github/ISSUE_TEMPLATE/01_bug_report.yml

```diff
@@ -19,7 +19,10 @@ body:
   - type: input
     attributes:
       label: "`semantic-release` version"
-      description: Version set in `package.json` `devDpendencies`.
+      description: |
+        The version executed in your pipeline, printed in the first line of output.
+        If not already using the latest version, please try upgrading to the latest version and refreshing your lockfile to see if the issue persists.
```

Comment:
we should more officially document our willingness to support old versions, but this is a start. we also get lots of support requests that could have been resolved by refreshing a lockfile when folks do decide to depend directly on semantic-release (whether by choice or because our recommendation isnt visible enough, or is inconsistent in our docs)